### PR TITLE
Add Python Package `tile_mate`

### PR DESCRIPTION
**Pull request by cmarshak**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/tile_mate/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.10
+    - setuptools >=60
```
Comment (jhkennedy) [inline recipes/tile_mate/meta.yaml:20]:
```suggestion
    - python-build
    - setuptools >=60
```
I think you might need this here

File: recipes/tile_mate/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+test:
+  imports:
+    - tile_mate
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  summary: Create rasters from global tiles
```
Comment (jhkennedy) [inline recipes/tile_mate/meta.yaml:45]:
```suggestion
  home: https://github.com/OPERA-Cal-Val/tile-mate
  summary: Create rasters from global tiles
```

**Review by jhkennedy — COMMENTED** [review]
I'm happy to be a maintainer!

**Comment by cmarshak** [comment]
@conda-forge-admin, please lint

**Comment by cmarshak** [comment]
@conda-forge-admin, please lint

**Comment by cmarshak** [comment]
@conda-forge/help-python, ready for review!

**Pull request merged**